.github/workflows/pypi.yml

```diff
@@ -16,10 +16,6 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: '3.8'
-    - name: Install dependencies
-      run: |
-        python -m pip install -U pip
-        python -m pip install setuptools wheel
```

Comment:
I’m not sure about the changes here, can we revert these?

Reply:
done